applications/social-networks.md

```diff
@@ -0,0 +1,17 @@
+---
+description: Machine translation for social media
+---
+
+# Social Networks
+
+Machine translation for **social networks** such as Facebook or Twitter is the translation of specific content chosen by the reader. The machine translation is used for gisting.
```

Comment:
Machine translation for **social networks** is integrated into social networks for gisting dynamic content on demand.

Social networks with a machine translation feature include:
- Facebook
- Twitter
- LinkedIn
- Instagram
- Sina Weibo
- YouTube